arangod/Cluster/ActionBase.cpp

```diff
@@ -148,25 +148,16 @@ std::shared_ptr<Action> ActionBase::getPostAction() {
 }
 
 
-// FIXMEMAINTENANCE: Code path could corrupt registry object because
```

Comment:
Put this comment back into the code.  The lock problem is not resolved by this change.  But resolution can wait since no one uses postAction's yet.